Python_Errors.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[3]:
+
+
+###Error corretion 1###
+
+seasons = ['Spring', 'Summer', 'Fall', 'Winter']
+print('My favorite season is ' + seasons[3])
+
+
+# In[7]:
+
+
+###Error correction 2###
+
+for number in range(10):
+    message = "message: "
+    # use a if the number is a multiple of 3, otherwise use b
+    if (number % 3) == 0:
+        message = message + "a"
+    else:
+        message = message + "b"
+    print(message)
+
+
+# In[30]:
+
+
+###Exceptions###
+
+x = list(input('Your name: '))
+for i in x:
+    try:
+        int(i)
+        raise Exception('Your name cannot include a number.')
+    except ValueError:
+        pass
+if ' ' in x:
+    raise Exception('Your name includes spaces, remove them.')
+if x[0].islower():
+    raise Exception("Your name cannot with lower letter.")
```

Comment:
Nevim, zda je to chyba uz v originale, ale vypadlo tu asi sluvko "start" ci tak.